sections/git_branch.zsh

```diff
@@ -12,6 +12,18 @@ SPACESHIP_GIT_BRANCH_PREFIX="${SPACESHIP_GIT_BRANCH_PREFIX="$SPACESHIP_GIT_SYMBO
 SPACESHIP_GIT_BRANCH_SUFFIX="${SPACESHIP_GIT_BRANCH_SUFFIX=""}"
 SPACESHIP_GIT_BRANCH_COLOR="${SPACESHIP_GIT_BRANCH_COLOR="magenta"}"
 
+# Show current commit SHA configuration
+SPACESHIP_GIT_BRANCH_SHOW_COMMIT="${SPACESHIP_GIT_BRANCH_SHOW_COMMIT=false}"
+SPACESHIP_GIT_BRANCH_COMMIT_SYMBOL="${SPACESHIP_GIT_BRANCH_COMMIT_SYMBOL=" ¢ "}"
+SPACESHIP_GIT_BRANCH_PREFIX_COMMIT="${SPACESHIP_GIT_BRANCH_PREFIX_COMMIT=""}"
+SPACESHIP_GIT_BRANCH_SUFFIX_COMMIT="${SPACESHIP_GIT_BRANCH_SUFFIX_COMMIT=""}"
+
+# Show tag configuration
+SPACESHIP_GIT_BRANCH_SHOW_TAG="${SPACESHIP_GIT_BRANCH_SHOW_TAG=false}"
+SPACESHIP_GIT_BRANCH_TAG_SYMBOL="${SPACESHIP_GIT_BRANCH_TAG_SYMBOL=" ŧ "}"
```

Comment:
There symbols are not properly rendered with powerline fonts. Could you share the unicode points of `_COMMIT_SYMBOL` and `TAG_SYMBOL` ?

Reply:
Sure @salmanulfarzy!

* `SPACESHIP_GIT_BRANCH_COMMIT_SYMBOL` (¢) is `\u00a2`
* `SPACESHIP_GIT_BRANCH_TAG_SYMBOL` (ŧ) is `\u0167`

Of course, this is only a suggestion...

Reply:
https://unicode-table.com/en/00A2/
https://unicode-table.com/en/0167/

Reply:
> SPACESHIP_GIT_BRANCH_COMMIT_SYMBOL (¢) is \u00a2
> SPACESHIP_GIT_BRANCH_TAG_SYMBOL (ŧ) is \u0167

They're different from symbols shown in screenshots. 

https://unicode-table.com/en/00A2/
https://unicode-table.com/en/0167/

Reply:
Yes, because I have customized it in my `~/.zshrc`:

```shell
# Git info
SPACESHIP_GIT_BRANCH_SHOW_COMMIT=true
SPACESHIP_GIT_BRANCH_COMMIT_SYMBOL=" \ue821 "
SPACESHIP_GIT_BRANCH_SHOW_TAG=true
SPACESHIP_GIT_BRANCH_TAG_SYMBOL=" \ue817 "
```
I guess it depends on what typography you're using. Mine is [Fantasque+Awesome+Powerline](https://github.com/ztomer/fantasque_awesome_powerline), based on [Fantasque Sans Mono](https://github.com/belluzj/fantasque-sans) project.